platforms/posix/src/px4/common/main.cpp

```diff
@@ -500,11 +500,11 @@ void register_sig_handler()
 	sigaction(SIGPIPE, &sig_pipe, nullptr);
 }
 
-extern "C" bool muorb_kill_slpi(void);
+// extern "C" bool muorb_kill_slpi(void);
```

Comment:
We need a way to conditionally add this line here so that it only runs when on the voxl target